app/models/note.rb

```diff
@@ -2,6 +2,8 @@ class Note < ActiveRecord::Base
   belongs_to :candidate
   belongs_to :user
   before_save :update_candidate
+  validates :comment, presence: true, length: { in: 10..2000 }
```

Comment:
Should provide some prompt in the UI to ensure that comment size is at least 10 characters long